backend/IDE.API/appsettings.json

```diff
@@ -21,5 +21,6 @@
     "DatabaseName": "FileStorageDb"
   },
   "RabbitMQ": "amqp://admin:admin@localhost:5672",
-  "ElasticSearch": "http://localhost:9200"
+  "ElasticSearch": "http://localhost:9200",
+  "emailApiKey": "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",
```

Comment:
NO keys in appsettings, remove it from the application settings file, RE-GENERATE a new key and change the logic to use environment variables instead. Also, be sure to update the DevOps Azure wiki page and notify other team members of the changes.